specification/compute/resource-manager/readme.md

```diff
@@ -0,0 +1,87 @@
+# Compute

```

Comment:
Not really, I've seen us use management before, but I just searched around in the Azure docs online and they seem to refer to it as Azure Resource Manager, so let's go with manager. Thanks!